src/server/connection.rs

```diff
@@ -618,6 +618,7 @@ impl Connection {
         }
         #[cfg(target_os = "linux")]
         clear_remapped_keycode();
+        release_modifiers();
```

Comment:
Suppose a scenario, there are two connections, one is pressing shift, the other connection is closed.